daemon/response.go

```diff
@@ -142,6 +142,8 @@ const (
 	errorKindSnapNeedsDevMode       = errorKind("snap-needs-devmode")
 	errorKindSnapNeedsClassic       = errorKind("snap-needs-classic")
 	errorKindSnapNeedsClassicSystem = errorKind("snap-needs-classic-system")
+
```

Comment:
Nitpick, you could have (probably) add this directly after the other `errorKind...` variables above without any ill effects on formatting.